source/features/unfinished-comments.tsx

```diff
@@ -0,0 +1,39 @@
+import select from 'select-dom';
+import cache from 'webext-storage-cache';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+const cacheKey = __filebasename + ':' + window.location.href;
+
+function init(): void {
+	if (document.body.dataset.visibilityListenerAdded) {
+		return;
+	}
+
+	const textareas = select.all('textarea');
+
+	document.addEventListener('visibilitychange', async () => {
```

Comment:
This function needs to be moved to the top level so you don’t need to use dataset. There’s no problem with just selecting the textareas every time.